samples/react-chat/src/App.tsx

```diff
@@ -26,15 +26,11 @@ const ChatListWrapper = memo(({ onSelected }: { onSelected: (e: GraphChat) => vo
     }
   ];
 
-  return (
-    <Get
-      resource="me/chats?$expand=members,lastMessagePreview&orderby=lastMessagePreview/createdDateTime desc"
-      scopes={['chat.read']}
-      cacheEnabled={false}
-    >
-      <ChatList menuItems={menus} buttonItems={buttons} onSelected={onSelected} />
-    </Get>
-  );
+  const onMessageReceived = () => {
+    console.log('PlexChatLog: Message received');
```

Comment:
actual remove the word plex since this is customer info - the final PR should not have customer specific info